dist/autoreviewcomments.user.js

```diff
@@ -1,31 +1,31 @@
-/** @preserve
```

Comment:
You probably changed the EOL terminator for the entire file. Don't do that

Reply:
Is it there in the final commit? I noticed the problem in the very first commit, and reverted it afterwards.

Reply:
If so, please squash your commits so the actual change is more apparent.